### PR TITLE
Fix #266

### LINKED ISSUE #266: Error in WineInstallationTest
Every unit test in WineInstallationTest fail. Here the trace.

com.playonlinux.wine.WineException: Wine has encountered a fatal error
	at com.playonlinux.wine.WineInstallation.run(WineInstallation.java:102)
	at com.playonlinux.wine.WineInstallationTest.testRun_RunWineVersionWithArgument_ProcessReturnsHelpMessage(WineInstallationTest.java:63)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
Caused by: java.io.IOException: Cannot run program "/home/magnus/Workspaces/PlayOnLinux/playonlinux/target/test-classes/com/playonlinux/wine/bin/wine" (in directory "/tmp"): error=13, Permission non accordée
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
	at com.playonlinux.wine.WineProcessBuilder.build(WineProcessBuilder.java:84)
	at com.playonlinux.wine.WineInstallation.run(WineInstallation.java:100)
	... 25 more
Caused by: java.io.IOException: error=13, Permission non accordée
	at java.lang.UNIXProcess.forkAndExec(Native Method)
	at java.lang.UNIXProcess.<init>(UNIXProcess.java:248)
	at java.lang.ProcessImpl.start(ProcessImpl.java:134)
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
	... 27 more

### PR DESCRIPTION
**Pull request by Tomahna**
Fix Issue #266 by adding a check when retrieving wine and wineServer. If the file isn't executable, give execute rights.

### CODE REVIEW

File: src/main/java/com/playonlinux/wine/WineInstallation.java [hunk 1]
```diff
@@ -50,29 +50,36 @@ private WineInstallation(WineInstallation.Builder builder) {
...
-
-
     // TODO
     public String fetchVersion() {
         return null;
     }
 
     private File fetchWineExecutablePath() {
-        return new File(binaryPath, "wine");
+        File wine = new File(binaryPath, "wine");
```
Comment (qparis) [inline src/main/java/com/playonlinux/wine/WineInstallation.java]: Can you create a separate private method please?

**Comment by Tomahna** [comment]
Done.

**Comment by qparis** [comment]
ok to test

**Comment by qparis** [comment]
I would call it setExecutablePermissions

**Pull request closed without merging**